src/main/resources/sprache/French.properties

```diff
@@ -1455,4 +1455,10 @@ ls.club.arena.income.terraces = Income Terraces
 ls.club.arena.income.basicseating = Income Basic seating
 ls.club.arena.income.seatsunderroof = Income Seats under roof
 ls.club.arena.income.seatsinvipboxes = Income Seats in VIP boxes
-ls.club.arena.income.total = Income Total
\ No newline at end of file
+ls.club.arena.income.total = Income Total
+
+#Training preview
+trainpre.fulltrain=Entrainement complet
```

Comment:
@yaute74  Translations should be done via PoEditor. Other files are generated on the fly at compile time so manually edited translation files would get lost.

Reply:
We must modify only the english version ?

Reply:
yes correct